MoneyLoverClone/Scenes/EventScreen/Cell/EventTableViewCell.swift

```diff
@@ -54,25 +87,43 @@ final class EventTableViewCell: UITableViewCell, NibReusable {
         return cashEvent
     }
     
-    private func estimateDay(endDate: Date) -> Int {
+    func estimateDay(endDate: Date) -> Int {
+        let calendar = Calendar.current
         let now = Date()
-        let estimateDay = Calendar.current.dateComponents([.day], from: now, to: endDate ).day ?? 0
+        let date1 = calendar.startOfDay(for: now )
```

Comment:
```suggestion
        let date1 = calendar.startOfDay(for: now)
```
tương tự các dòng dưới